config/airflow.dev.cfg

```diff
@@ -379,7 +379,7 @@ worker_log_server_port = 8793
 # information.
 # http://docs.celeryproject.org/en/latest/userguide/configuration.html#broker-settings
 # broker_url = sqla+mysql://airflow:airflow@localhost:3306/airflow
-broker_url = sqla+postgresql+psycopg2://postgres:example@db/airflow-db
+broker_url = redis://airflow-broker
```

Comment:
do we want to change the `result_backend` setting as well? Not sure what the impact would be, and `The use of a database is highly recommended`

Reply:
No. Using a relational database is best for this.